ui/urls.py

```diff
@@ -37,4 +37,8 @@
     url(r'^api/v0/upload_subtitles/$', views.UploadVideoSubtitle.as_view(),
         name='upload-subtitles'),
     url(r'^api/v0/', include((router.urls, 'models-api'))),
+
+    url(r'^api/v0/moira/user/(?P<username_or_email>[-\w]+$|[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)$',
```

Comment:
Are you sure this regex will cover all valid emails?  For instance I think "&" is a valid character in an email. Might be safer to use a less stringent regex here.

Reply:
I have updated this to a basic regex.